lib/ansible/modules/files/find.py

```diff
@@ -422,7 +424,7 @@ def main():
                 for fsobj in (files + dirs):
                     fsname = os.path.normpath(os.path.join(root, fsobj))
 
-                    if os.path.basename(fsname).startswith('.') and not params['hidden']:
+                    if os.path.basename(fsname).startswith(b'.') and not params['hidden']:
```

Comment:
b_fsname..... There could be other variables in this file that need the b_ prefix.... The advantage of doing it now is that the next person who has to look at this file will know at a glance whether they need to be working with byte strings or native strigns.  Right now, if I start, say in the ```pfilter``` function and see the byte string in the regex, I end up having to trace all the way back through the code to find out whether all the inputs to the ```f``` parameter are bytes or not.